best/plot.py

```diff
@@ -88,9 +89,9 @@ def plot_posterior(best_results: BestResults,
         ...                          color='avocado')
         >>> plt.show()
     """
-    samples = best_results.trace[var_name]
+    samples = best_results.trace.posterior[var_name]
```

Comment:
This doesn't work because the samples from the 4 chains are in different dimensions so the histogram looks messed up, so we need to flatten them like before:

```suggestion
    samples = arviz.extract(best_results.trace.posterior, var_names=var_name)
```